Intermittent integration test failures: test_filter_expressions_with_mismatching_firefox_versions, test_filter_expressions_with_matching_firefox_versions

Ran into this one [a few times](https://app.circleci.com/pipelines/github/mozilla/experimenter?branch=exp-1010-bucketing-in-preview) on a PR with apparently unrelated changes:
```
FAILED test_filter_expression_validation.py::test_filter_expressions_with_mismatching_firefox_versions
FAILED test_filter_expression_validation.py::test_filter_expressions_with_matching_firefox_versions
```
Both seemed to have an error message like:
```
selenium.common.exceptions.JavascriptException: Message: [Exception... "Component returned failure code: 0x80520012 (NS_ERROR_FILE_NOT_FOUND) [nsIXPCComponents_Utils.import]"  nsresult: "0x80520012 (NS_ERROR_FILE_NOT_FOUND)"  location: "JS frame :: [System Principal] :: <TOP_LEVEL> :: line 4"  data: no]
```
Two retries with failures, then the third retry passed:
* https://app.circleci.com/pipelines/github/mozilla/experimenter/8532/workflows/f2c1c0a3-f9d3-4a96-aa6e-026e9d34fb5c/jobs/22993
* https://app.circleci.com/pipelines/github/mozilla/experimenter/8532/workflows/84e5027c-7e70-46e2-8172-16fca478a8c4/jobs/22991

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-1011)
┆Issue Number: EXP-1011

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -61,7 +61,7 @@ jobs:
     machine:
       docker_layer_caching: true
       image: ubuntu-1604:202007-01 # Ubuntu 16.04, Docker v19.03.12, Docker Compose v1.26.1
-    resource_class: xlarge
+    resource_class: 2xlarge # using 2xlarge to mitigate firefox resource constraints
     working_directory: ~/experimenter
     steps:
       - run:

